### ISSUE TITLE
accessibility help form

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#1774]
Currently our accessibility help form is a link from the accessibility statement that goes to the contact page. 

We should add an accessibility help form to replace the contact link which is easy enough. However, it would be a step up if the form could some how auto-populate the page that the user was on when they needed help. Maybe this needs to be in the proud bar to capture the page the user is on.

The Marin accessibility expert brought up a fair point that it's several steps to find the accessibility page, fill out the form and then go back to enter the URL of the page you were on when you had a problem. So it would be nice if we had a form that was available from any page and let admins know which page the user was on when they were having issues. 

How heavy of a lift is something like this? @aschmoe 

I could see a feedback form like this being helpful beyond just accessibility for users to flag broken links, typos, etc.

**Comment by lukefretwell** [comment]
@kevindherman we should add a default accessible form to city template and make sure that they change the form contact email to theirs.

**Comment by lukefretwell** [comment]
We should explore looking at defaulting the tabs (service center and hover) to a low percentage of the primary color or just make them default to a low percentage black (ex: #efefef).
